doc/rules/basic/numeric_literal_separator.rst

```diff
@@ -0,0 +1,51 @@
+==================================
+Rule ``numeric_literal_separator``
+==================================
+
+Adds separators to numeric literals of any kind.
+
+Configuration
+-------------
+
+``override_existing``
```

Comment:
Since the rule is called `NumericLiteralSeparatorFixer`, it would be great (and seems easy to implements) to have both possibility to add or remove separators.

The only issue I see is the fact that `override_existing` seems useless for the `remove separator` usage.

Reply:
How would you build the option array for it?

Reply:
It could be option `strategy` with available values: `use_separator` and `no_separator`, the latter should be default IMHO. In terms of `override_existing` it probably should trigger warning when used along with `no_separator`.

Reply:
Strategies were introduced, default is `no_separator`.